config/manager/manager.yaml

```diff
@@ -102,3 +102,26 @@ spec:
       - name: webhook-tls
         secret:
           secretName: webhook-server-tls
+---
+kind: PodDisruptionBudget
+metadata:
+  name: controller-manager-pdb
+  namespace: default
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app.kubernetes.io/name: k8s-pdb-autoscaler
+      app.kubernetes.io/managed-by: kustomize
+---
+apiVersion: apps.mydomain.com/v1
+kind: PDBWatcher
+metadata:
+  name: controller-manager-pdb-watcher
+  namespace: default
+spec:
+  pdbName: controller-manager-pdb-pdb
```

Comment:
extra -pdb at the end here?

Reply:
Debating just having the pdbwatchers name always exactly match the pdbsname so we don't even need the pdbName field but haven't done it yet.

Reply:
if its only because you want to remove the field.. then I think its okay.. not sure what other reasons there could be to keep same name as pdb for watcher too

Reply:
There is some code to make sure there aren't two pdbswatchers on the same pdb (because who is resposible for scaling?) forcing the same name removes that code and generally simpler is better.